balboa-client/src/main/scala/com/socrata/metrics/collection/PreBufferQueues.scala

```diff
@@ -1,13 +1,13 @@
 package com.socrata.metrics.collection
 
-import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.{BlockingQueue, LinkedBlockingQueue}
 
 import com.socrata.metrics.components.MetricEntry
 
 trait PreBufferQueue {
-  def queue:SocrataAbstractQueue[MetricEntry]
+  def queue: BlockingQueue[MetricEntry]
```

Comment:
Uhm.  This was abstracted for a reason and I remember a good one.  There was no wheel reinvention, we just wanted to have a shared base class for different Queues....